src/latch_cli/services/launch/launch_v2.py

```diff
@@ -19,13 +23,126 @@
 from latch_sdk_config.latch import config
 from latch_sdk_gql.execute import execute
 
+ExecutionStatus = typing.Literal[
```

Comment:
```suggestion
ExecutionStatus: TypeAlias = typing.Literal[
```

Reply:
is this form of `Literal` supported on 3.9?